### ISSUE TITLE
Multicut - hide chunk boundary and able to select both sides of supervoxel that is split by boundary 

### ISSUE THREAD

**Issue by szichieh** [seung-lab/neuroglancer#499]
Priority: medium
Is it possible for the end users to select both halves of supervoxel that is cut by boundary in one click?
Chunk boundary is an irrelevant information for users/players during proofreading. Users shouldn't need to learn how to best use multicut nearby the boundary. Plus, it will increase success of cutting if red and blue annotations are not accidentally put on each side of a single supervoxel.

**Comment by akhileshh** [comment]
I think this can be handled easily by the backend, not trivial to implement in neuroglancer as far as I know.

The backend can put supervoxel edges with infinite affinity in the same group (sink/source) or simply ignore such edges in the input.

@nkemnitz thoughts?

**Comment by nkemnitz** [comment]
Agreed. The conflicts could be silently handled by the backend, resolving conflicting points in a simple, predefined way... e.g. red group wins. That's probably easiest to implement and maintains the current speed for adding split points, as it doesn't introduce any additional communication between graph server and NG. Disadvantage is that by silently reassigning segment IDs you might end up creating an unexpected or even impossible split that is inexplicable from the UI.

Other solution is to deliver the edges to NG and do the work there. That's more work (not sure how much), and @sdorkenw mentioned earlier that retrieving edges is a rather slow procedure.
But the possibilities with that are really cool: One could fetch the infinity-weighted edges to collect and highlight the supervoxel partners across chunk boundaries. And at least locally, we could even show the actual edges between supervoxel, allowing users to immediately see whether or not a potential merger in 2D is an actual merger (connected by active edge), or just a self-touch (deactivate edge).

**Comment by akhileshh** [comment]
I'm planning to look into using SSDs attached to nodes to store edges instead of buckets, that should reduce the latency.
With committed use discounts the cost is comparable to buckets.
